src/redturtle/prenotazioni/profiles/default/rolemap.xml

```diff
@@ -51,6 +51,14 @@
       <role name="Bookings Manager" />
     </permission>
 
+    <permission acquire="True"
+                name="redturtle.prenotazioni: Add BookingType"
+    >
+      <role name="Manager" />
+      <role name="Site Administrator" />
+      <role name="Owner" />
```

Comment:
Manager
Site Administrator
Bookings Manager
Owner

ask project managers if it's right or we also need to add Contributor

Reply:
No <bookings manager> here \cc @NicolaZivago